pallets/staking/Cargo.toml

```diff
@@ -0,0 +1,40 @@
+[package]
+name = 'pallet-staking'
+description = "Gluwa's offchain worker staking based on substrate frame staking"
+
+version.workspace = true
+authors.workspace = true
+edition.workspace = true
+license.workspace = true
+publish.workspace = true
+repository.workspace = true
+
+[package.metadata.docs.rs]
+targets = ['x86_64-unknown-linux-gnu']
+
+[dependencies]
+frame-benchmarking = { optional = true, workspace = true }
+frame-support = { workspace = true }
+frame-system = { workspace = true }
+log = { workspace = true }
+pallet-staking-substrate.workspace = true
+parity-scale-codec = { workspace = true }
+scale-info = { workspace = true }
+sp-arithmetic.workspace = true
+sp-runtime = { workspace = true }
+sp-std = { workspace = true }
+sp-staking.workspace = true
+
+[features]
+default = ['std']
+runtime-benchmarks = ['frame-benchmarking']
+std = [
+    'parity-scale-codec/std',
+    'scale-info/std',
+    'frame-support/std',
+    'frame-system/std',
+    'frame-benchmarking/std',
+    'pallet-staking-substrate/std',
+    'sp-std/std',
+]
+try-runtime = ['frame-support/try-runtime']
```

Comment:
Not using anything outside workspace for now but please add to `.github/dependabot.yml`